apps/init.sh

```diff
@@ -4,24 +4,30 @@ set -e
 
 kubectl create namespace horizontal-scaling
 kubectl create namespace vertical-scaling
-
+kubectl create ns otel-demo
```

Comment:
If I understand everything correctly, the `otel-demo` should only be installed when not running inside a codespace right? Could we run those commands conditionally then?